lms/djangoapps/program_enrollments/management/commands/tests/test_migrate_saml_uids.py

```diff
@@ -0,0 +1,60 @@
+"""
+Tests for the migrate_saml_uids management command.
+"""
+from __future__ import absolute_import
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from factory import LazyAttributeSequence, SubFactory
+from factory.django import DjangoModelFactory
+from lms.djangoapps.program_enrollments.management.commands import migrate_saml_uids
+from social_django.models import UserSocialAuth
+from student.tests.factories import UserFactory
+
+
+class UserSocialAuthFactory(DjangoModelFactory):
+    """
+    Foctory for UserSocialAuth records
+    """
+    class Meta:
+        model = UserSocialAuth
+    user = SubFactory(UserFactory)
+    uid = LazyAttributeSequence(lambda o, n: '%s:%d' % (o.slug, n))
+
+    class Params:
+        slug = 'gatech'
+
+
+class TestMigrateSamlUids(TestCase):
+    """
+    Test migrate_saml_uids command.
+    """
+    provider_slug = 'gatech'
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestMigrateSamlUids, cls).setUpClass()
+        cls.command = migrate_saml_uids.Command()
+
+    def _format_email_uid_pair(self, email, uid):
+        return '{email}:{uid}'.format(email=email, uid=uid)
+
+    def _format_slug_urn_pair(self, slug, urn):
+        return '{slug}:{urn}'.format(slug=slug, urn=urn)
+
+    def test_single_mapping(self):
```

Comment:
Can you also do a test case for multiple users?
And another test case for a user who do not have any social auth entries?
What about a test case where a user have multiple active social auth entries?